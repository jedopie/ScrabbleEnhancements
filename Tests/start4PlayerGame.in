1
4
JED
EL
JACK
Jon
