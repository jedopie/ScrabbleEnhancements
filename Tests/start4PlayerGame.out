Welcome to Scrabble!
-------------------
Menu
----
1. New Game
2. Load Game
3. Credits (Show student information)
4. Quit

Type "help" for further information

> 2 Player Mode: Enter '2'
3 Player Mode: Enter '3'
4 Player Mode: Enter '4'
> Starting a New Game 

Enter a name for Player 1 (Uppercase characters only)
> 

Enter a name for Player 2 (Uppercase characters only)
> 

Enter a name for Player 3 (Uppercase characters only)
> 

Enter a name for Player 4 (Uppercase characters only)
> Sorry, please enter a name with uppercase characters
